net/host.go

```diff
@@ -109,15 +123,14 @@ func NewHost(cfg *Config) (*Host, error) {
 		Port:                     cfg.Port,
 		KeyFile:                  cfg.KeyFile,
 		Bootnodes:                cfg.Bootnodes,
-		ProtocolID:               cfg.ProtocolID,
+		ProtocolID:               ChainProtocolID(cfg.Env),
 		ListenIP:                 cfg.ListenIP,
 		AdvertisedNamespacesFunc: h.advertisedNamespaces,
 	})
 	if err != nil {
 		return nil, err
 	}
 
-	log.Debugf("using base protocol %s", cfg.ProtocolID)
```

Comment:
no more logging the protocol ID? i think this is useful

Reply:
We were getting some logging of the protocol ID from the `go-p2p-net` package when the stream handlers are created, but I just added this one back a little further up.